Modules/vis/visualisations/common/vis.helper.js

```diff
@@ -41,38 +41,7 @@ var view =
   {
     this.start = ((new Date()).getTime())-(3600000*24*time);	//Get start time
     this.end = (new Date()).getTime();	//Get end time
-  },
-  
-  'round_interval':function(interval)
```

Comment:
Ok i saw the post.
So this is only showing in multigraph because it displays the previous value just before loading the new data asynchronously.
But the real reason is the underlay data that coming wrong. Mysql for example does not seems to have this problem.
I'd like the fix to be applied to the engine and not the graphic because it will bite us later on other usecases.
 Why not always return the previous available timestamp at the engine timestamp slot instead of averaging to to the requested start time ?